9.3/paper.tex

```diff
@@ -685,11 +685,12 @@ \subsubsection{Practical implications}
 cell-local arrays need to be resized for each
 cell (as has previously already been the case in the $hp$-context):
 \begin{c++}
-std::vector<double> local_rhs;
+Vector<double> local_rhs;
 for (const auto &cell : dof_handler.active_cell_iterators())
  {
    hp_fe_values.reinit(cell);
    local_rhs.resize(cell->get_fe().n_dofs_per_cell());
```

Comment:
`Vector` doesn't have `resize`. Should this be `reinit`?

Reply:
I will fix directly.